src/Stratis.Bitcoin.Features.Consensus/Rules/ConsensusRule.cs

```diff
@@ -17,8 +17,9 @@ public abstract class ConsensusRule
         /// <summary>Instance logger.</summary>
         public ILogger Logger { get; set; }
 
+        //TODO: should this be IConsensusRules?
         /// <summary>The engine this rule belongs to.</summary>
-        public ConsensusRules Parent { get; set; }
+        public ConsensusRules Parent { get; set; } 
```

Comment:
Plase remove space in the end of the line

Reply:
how did you even find that ?

Reply:
CTRL + R , CTRL + W